### PR TITLE
feat(dh): move keywords back below abstract, improve badges

### PR DESCRIPTION
**Pull request by jahow**
### Description

This PR moves the keywords back to their original place to make them more visible and accessible.

### Screenshots

image

### Quality Assurance Checklist

- [ ] Commit history is devoid of any _merge commits_ and readable to facilitate reviews
- [ ] If **new logic** ⚙️ is introduced: unit tests were added
- [ ] If **new user stories** 🤏 are introduced: E2E tests were added
- [ ] If **new UI components** 🕹️ are introduced: corresponding stories in Storybook were created
- [ ] If **breaking changes** 🪚 are introduced: add the `breaking change` label
- [ ] If **bugs** 🐞 are fixed: add the `backport <release branch>` label
- [ ] The [documentation website](docs) 📚 has received the love it deserves

### CODE REVIEW

**Comment by jahow** [comment]
Do not merge this yet, waiting for approval

**Review by Angi-Kinas — APPROVED** [review]
Thanks @jahow!
LGTM :rocket:

**Pull request merged**
